packages/fast-layouts-react/src/utilities/breakpoint-tracker.ts

```diff
@@ -117,8 +128,9 @@ class BreakpointTracker {
         }
 
         this.openRequestAnimationFrame = true;
-        window.requestAnimationFrame(this.update);
+
+        if (canUseDOM()) {
+            window.requestAnimationFrame(this.update);
+        }
     };
 }
-
-export default new BreakpointTracker();
```

Comment:
This currently removes BreakpointManager default singleton. This was only removed to add defaultBreakpoint as a constructor Param - is there a way to keep this in?

Reply:
This change would be breaking.

Reply:
Is there a way to keep in the default constructor (so it doesn't have to be breaking?)

Reply:
Is there a reason not to use a public getter and setter as we currently do to enable configuration of breakpoints? That would allow you to customize it for the instance without removing the singleton.